docs/outlook/autolaunch-store-options.md

```diff
@@ -0,0 +1,73 @@
+    ...
+```
+
+An end-user or admin can acquire add-ins through AppSource or the inclient store. If your add-in's primary scenario or workflow requires event-based activation, you may want to restrict your add-ins available to admin deployment. To enable that restriction, we can provide flight code URLs. Thanks to the flight codes, only end-users with these special URLs can access the listing. The following is an example URL.
+
+`https://appsource.microsoft.com/product/office/WA000000000?flightCodes=EventBasedTest1`
+
+Users and admins can't explicitly search for an add-in by its name in AppSource or the inclient store when a flight code is enabled for it. As the add-in creator, you can privately share these flight codes with organization admins for add-in deployment.
+
+> [!NOTE]
+> While end-users can install the add-in using a flight code, the add-in won't include event-based activation.
+
+## Specify a flight code
+
+You can specify the flight code you want for your add-in by sharing that information in the **Note for certification** when you're publishing your add-in.
+
+![Screenshot showing example request for flight code in Notes for certification screen during publishing process.](../images/outlook-publish-notes-for-certification.png)
+
+## Deploy add-in with flight code
+
+After the flight codes are set, you'll receive the URL from the app certification team. You can then share the URL with admins privately.
+
+To deploy the add-in, the admin can use the following steps.
+
+- Sign in to admin.microsoft.com or AppSource.com with your Microsoft 365 admin account. If the add-in has Single sign-on (SSO) enabled, global admin credentials are needed.
+- Open the flight code URL into a web browser.
+- On the add-in listing page, select **Get it now**. You should be redirected to the integrated app portal.
+
+## Unrestricted AppSource listing
+
+If your add-in doesn't use event-based activation for critical scenarios (that is, your add-in works well without automatic invocation), consider listing your add-in in AppSource without any special flight codes. If an end-user gets your add-in from AppSource, automatic activation won't happen for the user. However, they can use other components of your add-in such as a task pane or UI-less command.
+
+> [!IMPORTANT]
+> This is a temporary restriction. In future, we plan to enable event-based add-in activation for end-users who directly acquire your add-in.
+
+## Update existing add-ins to include event-based activation
+
+You can update your existing add-in to include event-based activation then resubmit it for validation and decide if you want a restricted or unrestricted AppSource listing.
+
+After the updated add-in has been approved, organization admins who have already deployed the add-in will receive an update message in the admin portal. The message advises the admin about the event-based activation changes. After the admin accepts the changes, the update will be deployed to end-users.
+
```

Comment:
We should add a screenshot with update notification here. Attaching the screenshot here. 

![MicrosoftTeams-image (24)](https://user-images.githubusercontent.com/49477715/125277675-5e4ced80-e32f-11eb-9f17-27a28f8415f8.png)